functional/platform/nspawn.go

```diff
@@ -211,10 +211,10 @@ func (nc *nspawnCluster) prepCluster() (err error) {
 		return
 	}
 
-	if !strings.Contains(stdout, "172.17.0.1/16") {
-		_, _, err = run("ip addr add 172.17.0.1/16 dev fleet0")
+	if !strings.Contains(stdout, "172.18.0.1/16") {
```

Comment:
If you have running docker daemon with the default configuration in CoreOS (`172.17.42.1`), these networks intersect and tests don't work.